legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-ansi/src/main/java/org/finos/legend/engine/persistence/components/relational/ansi/optimizer/LowerCaseOptimizer.java

```diff
@@ -38,6 +38,7 @@ public PhysicalPlanNode optimize(PhysicalPlanNode component)
         {
             Field field = (Field) component;
             field.setName(field.getName() != null ? field.getName().toLowerCase() : field.getName());
+            field.setAlias(field.getAlias() != null ? field.getAlias().toUpperCase() : field.getAlias());
             return field;
         }
```

Comment:
I think we need to handle alias for all Value types.
Rather than checking just for Field.
May be we can have another if block for :
```
if (component instanceof Value) {
     // setAlias here....
}
```

Reply:
Sure changed!